packages/paste-core/components/editable-code-block/src/EditableCodeBlockHeader.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import {Box, safelySpreadBoxProps, type BoxProps} from '@twilio-paste/box';
+import type {asTags} from '@twilio-paste/heading';
+
+export interface EditableCodeBlockHeaderProps extends Partial<Omit<HTMLHeadingElement, 'children'>> {
+  children: string;
+  element?: BoxProps['element'];
+  as?: asTags;
+}
+
+export const EditableCodeBlockHeader = React.forwardRef<HTMLHeadingElement, EditableCodeBlockHeaderProps>(
+  ({children, element = 'EDITABLE_CODE_BLOCK_HEADER', as = 'h3', ...props}, ref) => (
+    <Box
+      {...safelySpreadBoxProps(props)}
+      ref={ref}
+      element={element}
+      as={as}
+      backgroundColor="colorBackgroundInverseStrong"
+      color="colorTextInverse"
+      paddingY="space50"
+      paddingX="space70"
+      margin="space0"
+      borderTopLeftRadius="borderRadius20"
+      borderTopRightRadius="borderRadius20"
+      fontWeight="fontWeightSemibold"
+      fontSize="fontSize30"
+    >
+      {children}
+    </Box>
+  )
+);
+
+EditableCodeBlockHeader.displayName = 'EditableCodeBlockHeader';
```

Comment:
Suggestion: missing prop-types here. (I am starting to question their value though, mainly because we forget to update them and don't necessarily check that they are still valid)

Reply:
I think we should begin moving away from the run-time check of proptypes as Twilio is mainly a Typescript shop now